### PR TITLE
(DOCS-10465): Include blurb on different MongoDB editions

### PR DESCRIPTION
**Pull request by atsansone**
This change is [Reviewable](https://reviewable.io/reviews/mongodb/docs/3331)

### CODE REVIEW

**Comment by atsansone** [comment]
@MongoCaleb : This is a tricky ticket because we don't declare the differences between Community and Enterprise elsewhere. Maybe the [ticket](https://jira.mongodb.org/browse/DOCS-10465) is asking for something unnecessary, but I'd be curious to see if you think this covers the scope of the need and the ticket. [Staged.](https://docs-mongodbcom-staging.corp.mongodb.com/anthonysansone/DOCS-10465/tutorial/install-mongodb-on-windows.html#overview)

File: source/tutorial/install-mongodb-on-windows.txt [hunk 1]
```diff
@@ -13,36 +13,47 @@ Install MongoDB Community Edition on Windows
...
+The Windows installer (``.msi``) file includes all other
+software dependencies and automatically upgrades any previous version 
+of MongoDB that was installed using an ``.msi`` file.
 
-MongoDB Community Edition requires Windows Server 2008 R2, Windows 7, or
-later. The ``.msi`` installer includes all other software dependencies
-and will automatically upgrade any older version of MongoDB installed
-using an ``.msi`` file.
+To find which version, service pack, and build of Windows is  
+running on your host and the processor type of that host, enter the
```
Comment (MongoCaleb) [inline source/tutorial/install-mongodb-on-windows.txt]: > running on your host, as well as the processor type of the host, enter...

Comment (atsansone) [reply source/tutorial/install-mongodb-on-windows.txt]: Agreed. Fixed.

**Comment by atsansone** [comment]
@kay-kim : This is RFM for `master` and `v3.6`. It is only a clarification between editions. It can stay or go as you see fit.

**Pull request merged**